src/electron/future/main.ts

```diff
@@ -242,4 +244,14 @@ export async function main() {
 			await createMainWindow();
 		});
 	}
+
+	const apiKey = keyStore.get("openAiApiKey") || "none";
```

Comment:
you can add a second argument as default. Also I suggest using an emptyt string as it will pass boolean checks.

```suggestion
	const apiKey = keyStore.get("openAiApiKey", "");
```